### DISCUSSION TITLE
Great Lakes API

### DISCUSSION THREAD

**Discussion by kingsloi** [weather-gov/api#499] [General]
Hi, 

I run a little air quality sensor website in the Miller Beach neighbourhood of Gary, Indiana: https://millerbeach.community 

I’m in the process of securing a professional-grade air quality sensor, and I’m looking to update the website with info on the lake, too. 

Ideally, I’d love to grab any advisories for Gary/NWI/Chicagoland, but any/all of the info available about the lake/surf/waves would be great: 
https://www.weather.gov/greatlakes/

Lake Michigan claims the most lives of the Great Lakes https://glsrp.org/statistics/, so whatever I can do to make people aware of the lake, I’m about it!

**Comment by JJKraw** [comment]
You may know this already, but just in case...

All current advisories for Lake Michigan can be accessed here:
   https://api.weather.gov/alerts/active/area/LM

Similar fetches for the other lakes (LS, LH, LE, LO, plus LC for Lake St Clair and SL for the piece of St Lawrence River in the US).

There is also a region call to grab all of the above:
  https://api.weather.gov/alerts/active/region/GL

**Reply by kingsloi** [reply]
Hey thanks @JJKraw, 

no, I didn't know of that API! I played around with it and ended up implementing it, it works a treat!

https://millerbeach.community
